dask/dataframe/core.py

```diff
@@ -1663,6 +1663,20 @@ def count(self):
     def mean(self):
         return 1.0 * self.sum() / self.count()
 
+    def get_group(self, key):
+
+        def _get_group(df, gkey):
+            grouped = df.groupby(gkey)
+            if key in grouped.groups:
+                return grouped[self.column_info].get_group(key)
+            else:
+                # to create empty Series, which has the same
+                # dtype as the original
+                return df[0:0][self.column_info]
+
+        return self.df.map_partitions(_get_group, self.column_info,
+                                      self.index)
```

Comment:
Because the `_get_group` function is created within the function it probably doesn't have a consistent string representation and so won't `tokenize` consistently.  We should either define this function outside (adding a `key` argument to `_get_group` rather than using it as a closure) or we should add a token to the `map_partitions` call.